### PR TITLE
Added support for live photos 

### PR DESCRIPTION
**Pull request by smedic**
Improved loading speed
Added filters for filtering videos by date and duration

--------------
DO NOT MERGE 
(still testing)
--------------

### CODE REVIEW

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -169,6 +169,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = G3AXZ78R6P;
```
Comment (smedic) [inline example/ios/Runner.xcodeproj/project.pbxproj]: I ll remove these when I come back :)

**Review by jlalvarez18 — APPROVED** [review]
This looks good for now. I am in the process of updating the package to have better handling of downloading for the cloud so I'll bring in your work from here.

**Comment by Sauraus** [comment]
@smedic please rebase and resolve conflicts

File: example/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jlalvarez18) [reply example/ios/Runner.xcodeproj/project.pbxproj]: Removed

**Pull request merged**